app/server/utils/auth.py

```diff
@@ -406,6 +406,8 @@ def create_user_response_object(user, auth_token, message):
         'request_feedback_questions': request_feedback_questions(user),
         'default_feedback_questions': current_app.config['DEFAULT_FEEDBACK_QUESTIONS'],
         'transfer_usages': transfer_usages,
+        'forgiving_deduct': current_app.config['FORGIVING_DEDUCT'],
```

Comment:
adding config that turns some of the features associated with this PR on on the mobile app